app/helpers/alerts_helper.rb

```diff
@@ -1,3 +1,11 @@
 # frozen_string_literal: true
 module AlertsHelper
+  def alert_display
+    if alert.resolvable? == true && alert.resolved == true then
```

Comment:
Where is the `alert` object passed?

```ruby
if alert.resolvable? == true && alert.resolved == true then
```
The above line of code sounds imperative. You can simplify it as

```ruby
if alert.resolvable? && alert.resolved?
```